manifests/example/vddk-namespace.yaml

```diff
@@ -0,0 +1,5 @@
+# For VDDK, the namespace may be "kubevirt-hyperconverged" or "openshift-cnv"
```

Comment:
I changed this to more specifically reference the vddk-init sidecar, instead of just "VDDK". The importer code does not care what namespace it is installed in, it only cares about having a pre-defined namespace to hold the ConfigMap that references a real vddk-init container image. This comes from the existing UI-driven VMware import workflow - it seemed like a good idea to keep that convention instead of inventing something new:

https://docs.openshift.com/container-platform/4.3/cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.html#cnv-creating-vddk-image_cnv-importing-vmware-vm
https://github.com/kubevirt/web-ui-components/pull/534

I think the only other option is to have every VDDK datavolume CRD define an explicit reference to a vddk-init image.

Reply:
Right, the config map should be in the namespace that CDI is installed in. So in general there are 3 options:
* No HCO: cdi is the install namespace:
* OKD, with HCO: kubevirt-hyperconverged is the namespace CDI is installed in.
* OCP, with HCO: openshift-cnv is the namespace CDI is installed in.
utils.GetNamespace() will get the namespace CDI is installed in regardless of which combination above is used. So no need to hard code these.

Reply:
Oh okay, you were saying this whole vddk-init mechanism should be moved into the current CDI namespace. I can do that.

Reply:
Right, and util.GetNamespace() will find the correct name space for you, regardless of how CDI is installed.

Reply:
Okay, I replaced this with GetNamespace and updated the relevant documentation and tests.